src/cool_seq_tool/__init__.py

```diff
@@ -9,3 +10,9 @@
 logger.setLevel(logging.DEBUG)
 
 LOG_FN = "cool_seq_tool.log"
+
+
+try:
+    __version__ = version("cool_seq_tool")
+except PackageNotFoundError:
+    __version__ = "unknown"
```

Comment:
Might be good to follow what VRS-Python does for namespace cleanup 

```suggestion
try:
    __version__ = version("cool_seq_tool")
except PackageNotFoundError:
    __version__ = "unknown"
finally:
    del version, PackageNotFoundError
```